src/transformers/modeling_tf_utils.py

```diff
@@ -612,14 +612,24 @@ def set_input_embeddings(self, value):
         else:
             raise NotImplementedError
 
-    def get_output_embeddings(self) -> tf.keras.layers.Layer:
```

Comment:
Can we leave the default `get_output_embeddings` function for now as well?

Reply:
Method restored!